jobs/rootfs-builder.jpl

```diff
@@ -106,18 +115,42 @@ node("debos && docker") {
     }
 
     j.dockerPullWithRetry(docker_image).
-        inside(" --privileged --device /dev/kvm ") {
+        inside(" --privileged --device /dev/kvm -v /dev:/dev") {
 
         stage("Build") {
             timeout(time: 8, unit: 'HOURS') {
 	        build(config, arch, pipeline_version, kci_core, rootfs_type)
             }
         }
 
+        stage("scp") {
+            sshagent(credentials : ['jenkins-chromeos-storage-private-key']) {
+                sh(script: """\
+ssh \
+-o StrictHostKeyChecking=no \
+-p 22022 \
+chromeos@storage.chromeos.kernelci.org 'mkdir ~/images/rootfs/chromeos/${pipeline_version} || true'
```

Comment:
```suggestion
chromeos@storage.chromeos.kernelci.org 'mkdir -p ~/images/rootfs/chromeos/${pipeline_version} || true'
```
to solve issues like this:
```
mkdir: cannot create directory '/home/chromeos/images/rootfs/chromeos/crostest005': File exists
```

Reply:
Having -p is great idea, i missed that, thanks, added.